drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -64,6 +68,9 @@ private void applyCustomAttributes(Context context, AttributeSet attrs) {
             String copyFontPath = typedArray.getString(R.styleable.DropCapView_copyFontPath);
             setCopyFontType(copyFontPath);
 
+            int defaultNumberOfDropCaps = 1;
```

Comment:
Yea I could 😆  I did this to match the others and wasn't sure I wanted a constant that was only in one place to live for that long.